Traceback (most recent call last):
  File "<frozen runpy>", line 189, in _run_module_as_main
  File "<frozen runpy>", line 112, in _get_module_details
  File "C:\Users\francisco.contreras\Desktop\jupyter-projects\notebooks\MAYA-Modular\app\__init__.py", line 2, in <module>
    from app.tablas_busqueda import tablas_bp
  File "C:\Users\francisco.contreras\Desktop\jupyter-projects\notebooks\MAYA-Modular\app\tablas_busqueda.py", line 1, in <module>
    @tablas_bp.route('/generar_tabla', methods=['GET'])
     ^^^^^^^^^
NameError: name 'tablas_bp' is not defined
